deployment_config/dev_vars.yml

```diff
@@ -1,12 +1,7 @@
-AUTH_BASE: https://uat.hsesinfo.org
-AUTH_CLIENT_ID: ((AUTH_CLIENT_ID))
-AUTH_CLIENT_SECRET: ((AUTH_CLIENT_SECRET))
```

Comment:
We're providing values for the `AUTH_*` and `SESSION_SECRET` variables with the `--var` flags in the `cf push` command, so defining them here does nothing.